integration/js/checks/RemoteAudioCheck.js

```diff
@@ -14,11 +14,11 @@ class RemoteAudioCheck extends AppTestStep {
   }
 
   stepDescription() {
-    return 'Check the remote audio';
+    return `Check the remote audio for ${this.checkStereoTones ? 'stereo' : 'mono'} tone`;
   }
 
   metricName() {
-    return `Check the remote audio for ${this.checkStereoTones ? 'stereo' : 'mono'} tone`;
+    return `RemoteAudio${this.testType == 'AUDIO_ON' ? 'Enabled' : 'Disabled'}Check`
```

Comment:
nit: Could you please use strict equals "==="?

Reply:
Do we need a different metric for stereo?

Reply:
Updated. Added a different metric for stereo here and in play random tone step.